dataworks-admin-utils.tf

```diff
@@ -0,0 +1,34 @@
+resource "github_repository" "dataworks-admin-utils" {
+  name        = "dataworks-admin-utils"
+  description = "Contains dataworks administrative utilities"
+  auto_init   = true
+
+  allow_merge_commit = false
+  has_issues         = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+}
+
+resource "github_team_repository" "dataworks_admin_utils_dataworks" {
+  repository = "${github_repository.dataworks-admin-utils.name}"
+  team_id    = "${github_team.dataworks.id}"
+  permission = "push"
+}
+
+resource "github_branch_protection" "dataworks_admin_utils_master" {
+  branch         = "${github_repository.dataworks-admin-utils.default_branch}"
+  repository     = "${github_repository.dataworks-admin-utils.name}"
+  enforce_admins = false
+
+  required_status_checks {
+    strict = true
+    contexts = ["concourse-ci/dataworks-admin-utils-pr"]
```

Comment:
this won't exist, so remove this line

Reply:
The readme instructions don't say to remove that part? And there should be a PR job in the end? of os this a two-phase approach?

Reply:
the `contexts` need removing/emptying